is_sequent_provable.ml

```diff
@@ -0,0 +1,9 @@
+let check_proof_complete_with_exceptions request_as_json =
+    let sequent = Raw_sequent.sequent_from_json request_as_json in
+    if not (Clauses.provable_sequent_as_classical sequent) then false
+    else Phase.valid_sequent sequent;;
```

Comment:
The main question is: do we check provability with our "SAT-solver" first or with Phase models first?

Reply:
No precise idea. Since none of the code is optimized yet, let use ignore this point until we find it slow, and then we will try to understand where the bottleneck is.

Reply:
You're right, and we may never have such issue.